<!DOCTYPE html>
<html lang="ko">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>p 372 (예외 처리 구문 내부에서 return 사용하기 1 )</title>
    <script>
        function test () {
            try {
                alert('A 위치입니다.') 
                throw "예외 강제 발생" // throw 키워드로 예외를 강제로 발생
            }
            catch (exception) {
                alert('B 위치입니다.')
                return
            }
            alert('C 위치입니다.') // return 키워드를 사용해 함수를 벗어났으므로 출력하지 않는다.
        }

        test()
    </script>
</head>
<body>
    
</body>
</html>